ajax/libs/tipograf/1.0.2/tipograf.css

```diff
@@ -0,0 +1,110 @@
+/**
+ * tipograf - Lightweight CSS typography base for your next project
+ * @version v1.0.1
```

Comment:
@x09326 
I think the author just forgot to update the version number.
I found that there are some updates in `tipograf.css` in `v1.0.2` and also updated the version in `package.json`
https://github.com/mightyCrow/tipograf/commit/5e22c14171772b40fa8948b79a2ba088fbf48973